ElectronClient/gui/NoteEditor/NoteBody/AceEditor/AceEditor.tsx

```diff
@@ -534,10 +534,14 @@ function AceEditor(props: NoteBodyEditorProps, ref: any) {
 	}, [renderedBody, webviewReady]);
 
 	useEffect(() => {
-		if (props.searchMarkers !== previousSearchMarkers || renderedBody !== previousRenderedBody) {
+		if (props.searchMarkers.keywords.length) {
+			// find-in-note highlighting
 			webviewRef.current.wrappedInstance.send('setMarkers', props.searchMarkers.keywords, props.searchMarkers.options);
+		} {
+			// search words highlighting
+			webviewRef.current.wrappedInstance.send('setMarkers', props.highlightedWords, props.searchMarkers.options);
 		}
-	}, [props.searchMarkers, renderedBody]);
+	});
```

Comment:
Please take these changes out, we'll deal with them in the separate pull request.